libs/books/feature/src/lib/book-search/book-search.component.html

```diff
@@ -7,21 +7,21 @@
       placeholder="Search for books to add to your reading list"
       formControlName="term"
     />
-    <button mat-icon-button matSuffix>
+    <button type="search" aria-label = "Search Books" mat-icon-button matSuffix>
       <mat-icon>search</mat-icon>
     </button>
   </mat-form-field>
 </form>
 
 <ng-container *ngIf="searchTerm; else empty">
   <div class="book-grid">
-    <div class="book" data-testing="book-item" *ngFor="let b of books">
-      <div class="book--title">
+    <div class="book" data-testing="book-item" *ngFor="let b of this.books$|async">
+      <div tabindex="0" class="book--title">
         {{ b.title }}
       </div>
       <div class="book--content">
         <div class="book--content--cover">
-          <img src="{{ b.coverUrl }}" />
+          <img alt="Book Cover Page Details" src="{{ b.coverUrl }}" />
```

Comment:
Since we're already reading the book title out, adding this alt attribute is repetitive and unnecessary. Since we don't have any other helpful info to add here, we can add an empty alt attribute instead: https://www.w3.org/WAI/tutorials/images/decorative/